src/app/pages/home/activity/transaction-details/transaction-details.page.html

```diff
@@ -0,0 +1,24 @@
+<mat-toolbar color="primary" *transloco="let t">
+  <button routerLink=".." routerDirection="back" mat-icon-button>
+    <mat-icon>arrow_back</mat-icon>
+  </button>
+  <span>{{ t('TransactionDetails') }}</span>
+</mat-toolbar>
+<div *transloco="let t" class="page-content">
+  <ng-container>
+    <h4 class="top-time">{{ details.created_at | date: 'short' }}</h4>
+    <mat-card class="transaction-card">
+      <mat-card-content class="row">
+        <mat-label> {{ details.asset.id }}</mat-label>
+        <button [class]="details.status" mat-stroked-button disableRipple>
```

Comment:
Currently, the type of `details` does not contain `status` property. Please extend the type of `details` from `Transaction`.